teachable_machine.html

```diff
@@ -70,6 +70,8 @@ <h3>Inputs</h3>
         <dl class="message-properties">
             <dt>payload<span class="property-type">image buffer</span></dt>
                 <dd>A binary buffer of an image</dd>
+            <dt>reload<span class="property-type">bool</span></dt>
+                <dd>Set to any value to reload the model. In the same time, ignore image in payload (if any) and do not perform image classification.</dd>         
```

Comment:
I would say send a `true` value.
```suggestion
                <dd>Set to true to reload the model. In the same time, ignore image in payload (if any) and do not perform image classification.</dd>         
```